Jjainschigg doc changes (#36)
Mirantis/launchpad: docs/getting-started.md

Your hosts must be able to communicate with one another (and potentially, with users in the outside world) on their IP addresses, using many ports. Depending on your infrastructure and security requirements, this can be relatively simple to achieve for evaluation clusters. See [Networking Considerations](networking-considerations.md).

### Host configuration checklist

Hosts must be provisioned with:

* _Sufficient vCPU, RAM, and SSD storage_ &mdash; See [system requirements](system-requirements.md) and [host configuration](host-configuration.md) for details. For a small evaluation cluster, Manager and Workers can be provisioned with 2 vCPUs, 8GB RAM, with a 20GB SSD. On AWS EC2, for example, these specifications are met by a t3.large instance type, with SSD storage expanded to 20GB from the default 10GB.
* _A supported operating system_ &mdash; Docker Enterprise Manager nodes run on a supported Linux (see [system requirements](system-requirements.md)). Worker nodes may, alternatively, run on Windows Server 2019.

Hosts must be configured to allow:

> * _Access via SSH (or WinRC for Windows hosts):_ &mdash; See [system requirements](system-requirements.md) and [host configuration](host-configuration.md) for more info.

Reviewer: ```suggestion
* _Access via SSH (or WinRM for Windows hosts):_ &mdash; See [system requirements](system-requirements.md) and [host configuration](host-configuration.md) for more info.
```

## Set up Mirantis Launchpad CLI tool

UCP clusters may be deployed, managed and maintained with the Mirantis Launchpad ("**launchpad**") CLI tool. This tool is updated regularly, so make sure you are always using the latest version.

> NOTE: `launchpad` has built-in telemetry for tracking the usage of the tool. The telemetry data is used to improve the product and overall user experience. No sensitive data about the clusters is included in the telemetry payload.

Download and install the latest version of `launchpad` for the OS you are using below:

* [Download Launchpad](https://github.com/Mirantis/launchpad/releases/latest)
* Rename the downloaded binary as `launchpad` and move it to some dir in PATH and give it an execute permission.

> * With OSX you may also have to also allow Launchpad to be executed in the Security and Privacy settings.

Reviewer: ```suggestion
* With macOS you may also have to also allow Launchpad to be executed in the Security and Privacy settings.
```

INFO[0021] You can also download the admin client bundle with the following command: launchpad download-bundle --username <username> --password <password>
```

> By default, the admin username is `admin`. If you did not supply the password in with `cluster.yaml` or via the `installFlags` option like `--admin-password=supersecret`, the generated admin password is outputted in the install flow:

Reviewer: ```suggestion
By default, the admin username is `admin`. If you did not supply the password in with `cluster.yaml` or via the `installFlags` option like `--admin-password=supersecret`, the generated admin password will be displayed in the install flow:
```

Not native speaker, but "is outputted" seems wrong, maybe "is output"? Still sounds weird :)
